src/chain_spec.rs

```diff
@@ -136,7 +139,15 @@ fn general_testnet_genesis(
             code: general::WASM_BINARY.to_vec(),
             changes_trie_config: Default::default(),
         }),
-        asset: Some(GeneralConfig::AssetConfig {
+        asset: {
+
+            let mut file = File::open("~/src/data/currency_symbols.json").unwrap();
```

Comment:
If you want to read a file then with https://doc.rust-lang.org/nightly/std/fs/fn.read_to_string.html you'd only need `let contents = fs::read_to_string("path").unwrap();`

Reply:
I used Vladi's suggestion of having the vector of country symbols in the chain spec file

Reply:
But I have a question, in terms of how to convert the the Vec<&[u8; 3]> into Ticker format?

Reply:
You can use https://docs.polymesh.live/src/polymesh_primitives/ticker.rs.html#41-56 for that conversion.